agent/agent.go

```diff
@@ -303,7 +308,23 @@ func (a *Agent) runPostTaskCommands(ctx context.Context, tc *taskContext) {
 		} else {
 			tc.logger.Task().Infof("Finished running post-task commands in %v.", time.Since(start).String())
 		}
-		a.killProcs(tc)
+		grip.Info("Finished running post-task commands")
+	}
+}
+
+func (a *Agent) runPostGroupCommands(ctx context.Context, tc *taskContext, taskGroup *model.TaskGroup) {
+	grip.Info("Running post-group commands")
+	a.killProcs(tc)
+	defer a.killProcs(tc)
+	var cancel context.CancelFunc
+	ctx, cancel = a.withCallbackTimeout(ctx, tc)
+	defer cancel()
+	var err error
+	err = a.runCommands(ctx, tc, taskGroup.TeardownGroup, false)
+	if err != nil {
+		grip.Errorf("Error running post-task command: %v", err)
+	} else {
+		grip.Info("Finished running post-group commands")
```

Comment:
I'm inclined to just bake this into conditional logging/errorswrap statements

Reply:
fixed